code/game/objects/items/mixing_bowl.dm

```diff
@@ -107,7 +107,7 @@
 			dat = {"<B>[src] is empty</B><BR>"}
 		else
 			dat = {"<b>Ingredients:</b><br>[dat]"}
-		dat += {"<HR><BR> <A href='?src=[UID()];action=dispose'>Eject ingredients!</A><BR>"}
+		dat += {"<HR><BR> <A href='?src=[UID()];action=dispose'>Dispose ingredients!</A><BR>"}
```

Comment:
```suggestion
		dat += {"<hr><br> <a href='?src=[UID()];action=dispose'>Dispose ingredients!</a><br>"}
```

Reply:
Why is the standard hr/a instead of HR/A? I will try to change those in advance in future PRs if possible.

Reply:
standard practice for HTML is to use lowercase for elements